ow_faults/src/FaultInjector.cpp

```diff
@@ -16,7 +16,10 @@ FaultInjector::FaultInjector(ros::NodeHandle node_handle)
 {
   m_joint_state_sub = node_handle.subscribe("/_original/joint_states", 10, &FaultInjector::jointStateCb, this);
   // 'new' joint states topic where fault data is now publishes on
-  m_joint_state_pub = node_handle.advertise<sensor_msgs::JointState>("/joint_states", 10); 
+  m_joint_state_pub = node_handle.advertise<sensor_msgs::JointState>("/joint_states", 10);
+
+  m_dist_pitch_ft_sensor_sub = node_handle.subscribe("/_original/ft_sensor_dist_pitch", 10, &FaultInjector::distPitchFtSensorCb, this);
```

Comment:
Since it is used twice, consider putting "ft_sensor_dist_pitch" in a const string to avoid future mix ups. I could have done that with "joint_states" as well.

Reply:
addressed